### PR TITLE
python3Packages.grapheme: init at 0.6.0

### PR DESCRIPTION
**Pull request by Creator54**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality importing and using in python3 env
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/grapheme/default.nix [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+buildPythonPackage rec {
+  pname = "grapheme";
+  version = "0.6.0";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1jiwc3w05c8kh22s3zk7a8km8na3plqc5zimb2qcyxxy3grbkhj4";
+  };
```
Comment (jonringer) [inline pkgs/development/python-modules/grapheme/default.nix:12]: Pypi sdist is missing tests, please run the tests from github.

There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.

```

----------------------------------------------------------------------
Ran 0 tests in 0.000s
```

Comment (Creator54) [reply pkgs/development/python-modules/grapheme/default.nix:12]: added ``pythonImportsCheck = [ "grapheme" ];`` as v0.6.0 is only released on pypi.

Comment (fabaff) [reply pkgs/development/python-modules/grapheme/default.nix:12]: Perhaps we get tags.

https://github.com/alvinlindstam/grapheme/issues/18

Comment (fabaff) [reply pkgs/development/python-modules/grapheme/default.nix:12]: No response from upstream.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 154570` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python310Packages.grapheme</li>
    <li>python39Packages.grapheme</li>
  </ul>
</details>

**Review by fabaff — APPROVED** [review]
- [x] package path fits guidelines
- [x] package name fits guidelines
- [x] package version fits guidelines
- [x] `meta.description` is set and fits guidelines
- [x] `meta.license` fits upstream license
- [x] `meta.maintainers` is set
- [x] package builds (see https://github.com/NixOS/nixpkgs/pull/154570#issuecomment-1028162101)
- [ ] all depending packages build
- [x] tested on x86_64

Looks good to me.

**Pull request merged**